packages/zod-openapi/src/export.ts

```diff
@@ -0,0 +1,90 @@
+import { stringify as yamlStringify } from 'yaml'
+import type { OpenAPIHono } from './index'
+
+export interface ToDocOptions {
+  outDir?: string
+  format?: 'json' | 'yaml'
+  extension?: string
+}
+
+export interface FileSystem {
+  writeFile(path: string, data: string | Uint8Array): Promise<void>
+  mkdir(path: string, options: { recursive: boolean }): Promise<void | string>
+}
+
+export interface ToDocResult {
+  success: boolean
+  outDir: string
+  files: string[]
+  error?: Error
+}
+
+export const toDoc = async (
```

Comment:
I like the name `toDoc`, but the word `Doc` can be confusing in the case of OpenAPI. `toFiles` might be fine. Bu, any other good names?